### PR TITLE
Added NRIC, Organization and HealthWorker class. Made changes to Person class.

### PR DESCRIPTION
**Pull request by Lookuz**

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Person.java [hunk 1]
```diff
@@ -34,6 +35,21 @@ public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tag
         this.email = email;
         this.address = address;
         this.tags.addAll(tags);
+        this.nric = new Nric("S1234567Z"); //Placeholder NRIC
```
Comment (daviddl9) [inline src/main/java/seedu/address/model/person/Person.java]: Placing a placeholder value might not be a good idea without adding / modifying the existing tests. 

Also, perhaps we can use an `Optional<String> NRIC` to represent the NRIC?

Comment (Lookuz) [reply src/main/java/seedu/address/model/person/Person.java]: Ah but I was thinking cause NRIC should be a compulsory field due to the non-null and valid checks, and the placeholder is for enforcement of previous tests. For future implementations, we should be able to remove this original constructor and use the one that includes the NRIC.

**Comment by daviddl9** [comment]
Do update the testcases for the `Person` class, and the `PersonBuilder` class. 

Modification of the `PersonBuilder` class also has to be done to acocunt for the NRIC field. 

Adding NRIC to the `Person` object requires quite a bit of refactoring of the other classes and tests - which will affect the upcoming features that we will be working on as well. If this is not a priority right now, we can consider implementing the NRIC field in the subclasses first (i.e `HealthWorker` and `Patient`)

File: src/main/java/seedu/address/model/person/Person.java (hunk 1 above)
Comment (daviddl9) [reply src/main/java/seedu/address/model/person/Person.java]: Ahh I see. Okay!

**Pull request merged**